packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -2422,6 +2422,7 @@ export class StripeHelper {
     const {
       amount: paymentAmountOldInCents,
       currency: paymentAmountOldCurrency,
+      interval: productPaymentCycleOld,
```

Comment:
@chenba , I believe `productPaymentCycleOld` should be included here. Please review when you have a moment. Thank you!